zipkin-storage/cassandra/src/test/java/zipkin2/storage/cassandra/ITEnsureSchema.java

```diff
@@ -77,12 +90,19 @@
     try (CassandraStorage storage = CassandraStorage.newBuilder()
       .contactPoints(contactPoint.getHostString() + ":" + contactPoint.getPort())
       .ensureSchema(false)
+      .autocompleteKeys(asList("environment"))
```

Comment:
@zeagord here was the secret to causing us to figure out if we *really* work when schema isn't updated

Reply:
:) I totally forgot about this.